grpc/src/main/java/io/stargate/grpc/codec/cql/ValueCodecs.java

```diff
@@ -28,12 +28,23 @@
           new EnumMap<Type, ValueCodec>(Column.Type.class) {
             {
               put(Type.Ascii, new StringCodec(TypeCodecs.ASCII));
-              put(Type.Text, new StringCodec(TypeCodecs.TEXT));
-              put(Type.Varchar, new StringCodec(TypeCodecs.TEXT));
+              put(Type.Bigint, new BigintCodec());
+              put(Type.Blob, new BytesCodec());
+              put(Type.Boolean, new BooleanCodec());
+              put(Type.Counter, new BigintCodec());
+              put(Type.Date, new DateCodec());
+              put(Type.Double, new DoubleCodec());
+              put(Type.Float, new FloatCodec());
               put(Type.Int, new IntCodec());
-              put(Type.Uuid, new UuidCodec(TypeCodecs.UUID));
+              put(Type.Inet, new InetCodec());
+              put(Type.Smallint, new SmallintCodec());
+              put(Type.Text, new StringCodec(TypeCodecs.TEXT));
+              put(Type.Time, new TimeCodec());
+              put(Type.Timestamp, new BigintCodec());
```

Comment:
Timestamp uses `BigintCodec`?

Reply:
Are we saying a timestamp is only numeric and we don't support ISO-8601 format?

Reply:
We could, but my goal was to support the simplest type on the wire. The client-side would use simple conversions to convert a `java.time.Instant` (Java) or a `time.Time` (golang) to a `int64`. Those types support parsing ISO string formats.

If we support string formats for `timestamp` should we then support string representations for all other types?